tests/golden_tests/compile_file/error_messages.hvm

```diff
@@ -1,5 +1,6 @@
 data A = (A)
 data B = (B)
+data C = (B)
```

Comment:
Can you also test that this fails? (maybe in another file if the message doesn't show here.
```
data A = A
A = 0

main = A
```